### PR TITLE
Issue144

### PR DESCRIPTION
**Pull request by Bryar-Frank**
fixes issue #144

### CODE REVIEW

**Comment by constantinol** [comment]
testing email notifs

File: _sources/Strings/Exercises.rst [hunk 1]
```diff
@@ -105,36 +105,28 @@ Exercises
...
-         
-              from unittest.gui import TestCaseGui
-
-              class myTests(TestCaseGui):
-                  def testOne(self):
-                      string1 = "e"
-                      string2 = "eieio"
-                      string3 = "eeeeeeeeeeee"
-                      string4 = "elephant"
-                      self.assertEqual(count(string1), 1, "Just one e")
```
Comment (bnmnetp) [inline _sources/Strings/Exercises.rst:128]: somehow these extra tests did not get added to the final version.

Comment (Bryar-Frank) [reply _sources/Strings/Exercises.rst:128]: I just fetched upstream again to update this branch, did that fix it?

Comment (Bryar-Frank) [reply _sources/Strings/Exercises.rst:128]: looks like it didn't

Comment (Bryar-Frank) [reply _sources/Strings/Exercises.rst]: This has been fixed thanks to Brad's help to update my master branch

**Comment by Bryar-Frank** [comment]
This was a duplicate

**Pull request closed without merging**